### PR TITLE
Fix LTR end to end notebook (Fixes 100)

### PR DESCRIPTION
**Pull request by softwaredoug**
Saves the product judgments file, and other fixes to the ch 11 end to end notebook

### CODE REVIEW

**Comment by treygrainger** [comment]
@softwaredoug I'm still getting a failure reading the product_judgements file. Does it need to be committed here?: https://github.com/ai-powered-search/retrotech

File: docker/data-science/notebooks/ch11/a.end-to-end-auto-ltr.ipynb [hunk 1]
```diff
@@ -1000,11 +1013,11 @@
       "Creating 'products' collection\n",
       "Status: Success\n",
       "Del/Adding LTR QParser for products collection\n",
-      "<Response [200]>\n",
-      "Status: Success\n",
+      "<Response [400]>\n",
+      "Status: Failure; Response:[ {'responseHeader': {'status': 400, 'QTime': 1}, 'errorMessages': [\"error processing commands, errors: [{delete-queryparser=ltr, errorMessages=[NO such queryParser 'ltr' ]}], \\n\"], 'WARNING': 'This response format is experimental.  It is likely to change in the future.', 'error': {'metadata': ['error-class', 'org.apache.solr.api.ApiBag$ExceptionWithErrObject', 'root-error-class', 'org.apache.solr.api.ApiBag$ExceptionWithErrObject'], 'details': [{'delete-queryparser': 'ltr', 'errorMessages': [\"NO such queryParser 'ltr' \"]}], 'msg': \"error processing commands, errors: [{delete-queryparser=ltr, errorMessages=[NO such queryParser 'ltr' ]}], \", 'code': 400}} ]\n",
```
[the remark this answers is not in the document]
Comment (softwaredoug) [reply docker/data-science/notebooks/ch11/a.end-to-end-auto-ltr.ipynb:1017]: Let me double check

**Comment by softwaredoug** [comment]
Let me do a clean rebuild again, but the file should be written on this cell:

```
judgments_to_file(open('data/product_judgments.txt', 'w'), judgments)
```

File: docker/data-science/notebooks/ch11/a.end-to-end-auto-ltr.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (softwaredoug) [reply docker/data-science/notebooks/ch11/a.end-to-end-auto-ltr.ipynb:1017]: This error can be safely ignored. I can add a note in the notebook.

**Review by treygrainger — APPROVED** [review]
All is working now. Thanks @softwaredoug !

**Pull request merged**